woodwork/tests/schema/test_schema_column.py

```diff
@@ -109,6 +107,32 @@ def test_get_column_dict_params():
     assert column.get('metadata') == {'created_by': 'user1'}
 
 
+def test_get_column_dict_null_params():
+    empty_col = {
+        'logical_type': None,
+        'semantic_tags': set(),
+        'description': None,
+        'metadata': {}
+    }
+    assert _get_column_dict() == empty_col
+
+    just_tags = _get_column_dict(semantic_tags={'numeric', 'time_index'})
+    assert just_tags.get('logical_type') is None
+    assert just_tags.get('semantic_tags') == {'numeric', 'time_index'}
+
+    just_tags = _get_column_dict(logical_type=Integer)
+    assert just_tags.get('logical_type') == Integer
+    assert just_tags.get('semantic_tags') == set()
+
+    error = "Cannot use standard tags when logical_type is None"
+    with pytest.raises(ValueError, match=error):
+        _get_column_dict(semantic_tags='categorical', use_standard_tags=True)
+
+    error = "semantic_tags for None must be a string, set or list"
```

Comment:
Think we should update this error message to only include the "for None" part if a name has been defined, otherwise just "semantic_tags must be a string, set or list"?

Reply:
Yes, that seems like a better way to handle this error message